tests/small/LandIce/FO_AIS/inputML1.yaml

```diff
@@ -20,8 +20,9 @@ ANONYMOUS:
         Cubature Degree: 3
         Side Set Name: basalside
         Basal Friction Coefficient:
-          Type: Given Field
-          Given Field Variable Name: basal_friction
+          Type: Power Law
```

Comment:
I wonder if we should keep `Given Field` (or 'Prescribed Field' maybe) as a valid option. We could simply have:
```
Basal Friction Coefficient:
  Type: Prescribed Field
  Field Name: basal_friction
```
Internally, we could still treat it as a Power Law with exponent=1.0, N=One, and muType=FIELD. But the input file would probably be less confusing for those interested in prescribing a field. It is kind of silly to have to specify a power exponent just so that we get rid of the power law altogether...

Reply:
We could, but we'll be adding yet another case. I think Matt and Trevor are fine always using muField also in the linear case.

Reply:
As you say it would be trivial to implement. But it would be used only in standalone Albany anyway, as that condition is hardcoded as the default one in the MPAS Interface as well.